crawler/src/main/resources/application.conf

```diff
@@ -0,0 +1,27 @@
+akka {
+  quartz {
+    defualtTimezone = "Europe/Warsaw"
+
+    schedules {
+      EverydayMidnight {
+        description = "Everyday at midnight"
+        expression = "0 0 0 * * ?"
+      }
+    }
+  }
+}
+
+bootstrap {
+  servers = "localhost:9092"
+}
+
+crawler {
+  http {
+    timeout = 10s
+    max.parallelism = 30
+  }
+}
+
+kafka {
+  topic = "newspaper-content"
+}
```

Comment:
I think in the long run we shouldn't be using cron-like scheduling, but fetching pages in fixed intervals distributed throughout the day. Until then yes, I had to change that line to `0 * * * * ?` to test if it works, so `environment.conf` would be double useful.